### DISCUSSION TITLE
Wrong position after opening tabs in product page [mobile view]

### DISCUSSION THREAD

**Discussion by sdomeradzki** [breezefront/community#11] [1. Help]
Hi,

When I choose any option from tabs, page not scroll correct. Always I have to scroll up, to start header of each tabs.
Please help.

[https://youtube.com/shorts/QX8vkOTnW-g?feature=share](https://youtube.com/shorts/QX8vkOTnW-g?feature=share)

**Comment by vovayatsyuk** [comment]
Thanks for reporting! Just fixed in the repo: https://github.com/breezefront/module-breeze/commit/df0a933060b2764b134fc057c8fd555ca54e517e

The fix will be included in the next release.
